projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -247,12 +252,15 @@ private function prepare_menu_item( array $menu_item ) {
 			$url                = $first_submenu_item[2];
 		}
 
+		$menu_identifier = $menu_item[5];
```

Comment:
Is this always guaranteed to be set? I noted there are several default menu items like the first separator which does not have a value in the fifth position: https://core.trac.wordpress.org/browser/trunk/src/wp-admin/menu.php?rev=49837#L62

Reply:
Well, it neither has a value in the sixth position, and we're always reading it a few lines below for getting the icon, so I guess nothing happens?